content/_index.md

```diff
@@ -20,10 +20,11 @@ imagealt = "Website banner of PyLadiesCon 2024 Online. 6th-8th December | Multi-
       </p>
       <div class="social w-100 mx-auto pt-3 text-start">
         <a href="https://fosstodon.org/@pyladiescon"><i class="fab fa-mastodon fa-2x px-2" aria-hidden="true"></i></a>
-        <a href="https://bsky.app/profile/pyladiescon.bsky.social"><i class="fab fa-bluesky fa-2x px-2" aria-hidden="true"></i></a>
+        <a href="https://bsky.app/profile/pyladiescon.bsky.social"><i class="fas fa-cloud fa-2x px-2" aria-hidden="true"></i></a>
```

Comment:
Thanks for this change, I will fix the fontawesome version in order for the icon to be displayed though!